src/lib.rs

```diff
@@ -35,11 +37,140 @@ mod tests;
 /// OSM id of object
 pub type ObjId = i64;
 
-/// Latitude
-pub type Lat = f32;
+/// How many nanodegrees are represented by each unit in [`Lat::inner()`].
+/// We use the same internal precision as OpenStreetMap.org, 100 nanodegrees.
+pub const COORD_PRECISION_NANOS: i32 = 100;
```

Comment:
I don't really expect these to change, I just extracted them into named constants because I think they are easier to understand than having magic number sprinkled around.